library/globals.inc.php

```diff
@@ -240,15 +241,41 @@ function gblTimeZones()
             '__default__',
             xl("Select the default font size"),
         ),
+        
+        'vertical_responsive_menu' => array(
+            xl('Responsive Vertical Menu Style for Tabs (need to logout/login after changing this setting)'),
+            array(
+                '736' => xl('iPhone 6/7/8 Plus') . " - " . attr('736 X 414 px'),
+                '740' => xl('Galaxy S9/S9 Plus') . " - " . attr('740 X 360 px'),
+                '812' => xl('iPhone X/XS') . " - " . attr('812 X 375 px'),
+                '896' => xl('iPhone XR/XS Max') . " - " . attr('896 X 414 px'),
+                '1024' => xl('iPad/iPad Mini, XGA') . " - " . attr('1024 X 768 px'),
+                '1112' => xl('iPad Pro 10.5 inches') . " - " . attr('1112 X 834 px'),
+                '1280' => xl('Kindle Fire HDX, Laptop MDPI , WXGA') . " - " . attr('1280 X 800 px'),
+                '1336' => xl('iPad Pro 12.5 inches') . " - " . attr('1336 X 1024 px'),
+                '1366' => xl('HD') . " - " . attr('1366 X 768 px'),
+                '1440' => xl('Laptop HiDPI, WXGA+') . " - " . attr('1440 X 900 px'),
+                '1600' => xl('HD+') . " - " . attr('1600 X 900 px'),
+                '1680' => xl('WSXGA+') . " - " . attr('1680 X 1050 px'),
+                '1920' => xl('FHD, WUXGA') . " - " . attr('1920 X 1080').", ". attr('1920 X 1200 px'),
+                '2048' => xl('QWXGA') . " - " . attr('2048 X 1152 px'),
+                '2560' => xl('QHD') . " - " . attr('2560 X 1440 px'),
+                '3840' => xl('4K UHD') . " - " . attr('3840 X 2160 px'),
```

Comment:
I bet the issue is the attr() calls here since I don't think this is defined in the install class. Remove the attr() calls and I bet the demo farm will then work. Plus should not be using attr anyways since these are not variables.

Reply:
for example, above line should just be:
```
'3840' => xl('4K UHD') . " - 3840 X 2160 px",
```